src/App.tsx

```diff
@@ -1,61 +1,123 @@
+import { ChangeEvent, FormEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodosList } from './types/TodoList';
+
+const combineTodosAndUser = todosFromServer.map(todo => (
+  {
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId),
+  }
+));
 
 export const App = () => {
+  const [title, setTitle] = useState<string>('');
+  const [choosedUser, setChoosedUser] = useState<string>('');
+  const [todos, setTodos] = useState<TodosList>(combineTodosAndUser);
+  const [titleError, setTittleError] = useState<boolean>(false);
+  const [choosedUserError, setChoosedUserError] = useState<boolean>(false);
```

Comment:
don't add type for primitive
```suggestion
  const [title, setTitle] = useState('');
  const [choosedUser, setChoosedUser] = useState('');
  const [todos, setTodos] = useState<TodosList>(combineTodosAndUser);
  const [titleError, setTittleError] = useState(false);
  const [choosedUserError, setChoosedUserError] = useState(false);
```